misc/python/materialize/mzcompose/services.py

```diff
@@ -1050,13 +1049,13 @@ def __init__(
         )
         toml.dump(
             {
-                "current_profile": "default",
+                "profile": "default",
                 "profiles": {
                     "default": {
-                        "email": username,
```

Comment:
So authentication no longer requires a username, just the app_password?

Reply:
`app_password` is enough. The CLI retrieves the username from Frontegg.